integration_test/cases/client_test.exs

```diff
@@ -138,9 +138,6 @@ defmodule ClientTest do
       end,
       {:idle, :new_state},
       {:idle, :new_state},
-      fn(_, _, _, _) ->
-        throw(:oops)
-      end,
```

Comment:
Same here. We never get to invoke this. So I would say this whole test is useless, because the scenario "when client timeout and then crashes" is no longer possible.